08-single-page-web-apps/tdd/readme.md

```diff
@@ -44,137 +44,156 @@ When it comes to development in teams, both the benefits and the potential pitfa
-  });
-});
+```bash
+$ mkdir spec
 ```
 
-Does any of it look familiar? *Yes*
-* What similarities does it have with RSpec?
-  * `describe`, `it`, `expect`
-  * Fortunately, we get to work with the same testing syntax as we do in Ruby.
-* Look at them side-by-side...
-
-![ruby js test side by side](img/ruby-js-side-by-side.png)
+Inside that new folder, create a file named after your model, followed by "dash-spec":
 
-## Let's get to work (20 / 55)
+```bash
+$ touch spec/snowman-spec.js
+```
 
-### Create your JS files.
+Now, run:
 
-Switch into your `sandbox` folder and ...
 ```bash
-$ mkdir snowman-builder
-$ cd snowman-builder
-$ touch snowman.js
-$ mkdir spec #our testing folder
-$ touch spec/snowman-spec.js # our test file
+$ jasmine-node spec
 ```
 
-### Think.
+You should get something like:
 
-Before we write any of our snowman code, we're going to create a test. And before we create a test, we're going to think: What do I want my snowman builder to do? What should the snowmen it creates be like?
+```
+$ jasmine-node spec
 
-```js
-//I want my snowman builder to create a snowman object
-//My winter wonderland is a friendly place, so I want each snowman to have a name
-//In order for it to really be a snowman, it needs to have a carrot nose.
-//It also needs stick arms.
-//If the snowman is named Olaf, he should like warm hugs.
+
+Finished in 0.001 seconds
+0 tests, 0 assertions, 0 failures, 0 skipped
 ```
 
-### Outline.
+## Adding the tests
+
+Now, paste the tests you wrote earlier into the `-spec.js` file you created.
+
+If you run `jasmine-node` at this point, you'll just get an error. We need to properly format the code for Javascript.
+
+`describe` and `it` are actually functions, just like, say, `console.log()`. They both take a string as their first argument. We'll make `describe` take a function as its second argument.
+
+For the snowman builder, the result would be this:
 
 ```js
-// spec/snowman-spec.js
+describe( "A snowman", function(){
+  it("should have a name");
+  it("should have a carrot nose");
+  it("should have stick arms");
+  describe( "a snowman named Olaf", function(){
+    it("should like warm hugs");
+  });
+});
+```
+
+If I run this, I now get:
+
+```
+$ jasmine-node spec
+....
 
-// Not unlike Ruby, `require` lets us reference `snowman.js` in our spec file.
+Finished in 0.009 seconds
+4 tests, 0 assertions, 0 failures, 0 skipped
+```
+
+Just like in RSpec, each green dot indicates a passing test.
+
+If I add the `--verbose` flag, I get:
 
-var snowman = require( "../snowman" );
 ```
+$ jasmine-node spec --verbose
+
+A snowman - 4 ms
+    should have a name - 1 ms
+    should have a carrot nose - 0 ms
+    should have stick arms - 0 ms
+
+    a snowman named Olaf - 0 ms
+        should like warm hugs - 0 ms
+
+Finished in 0.01 seconds
+4 tests, 0 assertions, 0 failures, 0 skipped
+```
+
+## You Do: Update your code so that it has the proper syntax
+
+It should run successfully when you're done.
```

Comment:
After re-reading the actual file (instead of this confusing diff), I believe you want me to update my "specs" so that they have the proper syntax.  "It" still feels like a strange pronoun here.  Also, "running successfully" is a strange expectation, there are no assertions. Suggest:  "Update your specs so that they have the proper syntax", "You should see a successful run."